rainier-core/BUILD

```diff
@@ -8,15 +8,12 @@ load(
 scala_library(
   name = 'rainierCore',
   deps = [
-    '//.rainier-shaded-asm:asmShaded',
-    '//.rainier-shaded-asm:asmTreeShaded',
+    '//external:jar/com/google/code/findbugs/jsr305',
     '//external:jar/com/google/flogger/flogger',
-    
-  ],
-  runtime_deps = [
```

Comment:
It looks like these runtim_deps got dropped. Is it possible this file was edited by hand before?

Reply:
whoa, good catch. Dunno, but I'll just add them back.

Reply:
Running `sbt bazelGenerate` on current develop, I get https://github.com/stripe/rainier/compare/mio/no-https which also deletes those deps. I know shaded-asm has some complicated history, did we intentionally change this? (cc: @avi-stripe)